app/helpers/manageiq/consumption/time_converter_helper.rb

```diff
@@ -10,7 +10,7 @@ module ManageIQ::Consumption
   module TimeConverterHelper
     VALID_INTERVAL_UNITS = %w(hourly daily weekly monthly yearly).freeze
 
-    def self.number_of_intervals(period:, interval:, calculation_date: Time.now, days_in_month: nil, days_in_year: nil)
+    def self.number_of_intervals(period:, interval:, calculation_date: Time.now.utc, days_in_month: nil, days_in_year: nil)
```

Comment:
Time.current=?

Reply:
Rubocop offense when time.now... or time.zone.now or other...I set utc. Which is better?

Reply:
Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.to_i, Time.now.to_f instead

Reply:
@gtanzillo @lpichler In your experience, what is the best way to use this?
How it is done in the old code?
Should we merge this and open an issue?